site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -422,13 +422,59 @@ The fields above do the following:
   - `LowerOrNewerEqualPriority`: only preempt Workloads in the ClusterQueue that either have a lower priority than the pending workload or equal priority and are newer than the pending workload.
 
 Note that an incoming Workload can preempt Workloads both within the
-ClusterQueue and the cohort. Kueue implements heuristics to preempt as few
-Workloads as possible, preferring Workloads with these characteristics:
+ClusterQueue and the cohort.
+
+### Preemption Algorithm overview
+
+An incoming Workload, which does not fit within the free capacity, is eligible
+for preemptions, within a flavor of the target queue, when one of the following
+is true:
+- the requests of the Workload are below the flavor's nominal quota, or
+- `borrowWithinCohort` is enabled.
+
+Kueue implements heuristics to preempt as few Workloads as possible, preferring
+Workloads with these characteristics:
 
 - Workloads belonging to ClusterQueues that are borrowing quota.
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+Below we present a more detailed description of the algorithm.
+
+#### Candidates
+
+The list of preemption candidates is compiled from Workloads within the Cluster
+Queue satisfying the `withinClusterQueue` policy, and Workloads within the
+cohort which satisfy the `reclaimWithinCohort` policy.
+
+The list of candidates is sorted based on the following preference checks for
+tie-breaking:
+- Workloads from other borrowing queues in the cohort,
+- Workloads with the lowest priority,
+- Workloads which got admitted the most recently.
```

Comment:
We should sync the list at L438 with this, and maybe louse one of them.

Just removing L438-L441 can be the best.

Reply:
Indeed, let me tentatively drop  L438-L441, and moved up as intro for the "Preemption Algorithm overview" section.